go/test/endtoend/backup/vtctlbackup/backup_utils.go

```diff
@@ -1299,6 +1300,13 @@ func TestReplicaRestoreToPos(t *testing.T, replicaIndex int, restoreToPos replic
 
 	require.False(t, restoreToPos.IsZero())
 	restoreToPosArg := replication.EncodePosition(restoreToPos)
+	assert.Contains(t, restoreToPosArg, "MySQL56/")
```

Comment:
Any reason not to use `DecodePositionMySQL56()` here instead? Even better if we move it after the rand block.

Reply:
That was intentional: I want to do a raw test of the encoded string, rather than an implicit conversion to a construct where the `MySQL56` may be added implicitly.